app/code/Inchoo/Sample01/Controller/Custom/Redirect.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inchoo\Sample01\Controller\Custom;
+
+use Magento\Framework\App\ActionInterface;
+use Magento\Framework\Controller\ResultFactory;
+
```

Comment:
opis klase nek vam uđe u praksu, što klasa hendla točno.. ako previše toga hendla za opis znači da trebate dvije klase. dodati na svim klasama

Reply:
ne treba stavljati doc block komentar ako se ne piše nešto specifično. Čak i novi code check prijavljuje grešku ako je default PHPStorm komentar. Bolje ostaviti bez ako se nema ništa pametno napisati